src/Components/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import '../CSS/Nav.css'
+
+const Nav = () => {
```

Comment:
I like the ideas you've included here including favoriting and sorting functionality in the Navigation here.  The only question I have is do you plan to add functionality for favoriting, sorting, and logging in/out, etc?  If not, I would recommend not including non-functioning elements as it can be a bit confusing for the end user.